### ISSUE TITLE
Yolo loss.

### ISSUE THREAD

**Issue by Nazarko12345** [learnml-today/object-detection-with-pytorch#13]
- We apply sigmoid in last layer of yolo to get coords (0,1) and probabilty of classes.Why you apply sigmoid in loss one more time ?
From code :
```
class_prediction_loss = ((torch.sigmoid(pred) - torch.sigmoid(target)) * obj_mask)[:, :, self.B*5:].pow(2).sum()
noobj_loss = self.lambda_noobj * ((torch.sigmoid(pred) - torch.sigmoid(target)) * noobj_mask)[:, :, 4:self.B*5:5].pow(2).sum()
obj_loss = ((torch.sigmoid(pred) - torch.sigmoid(target)) * responsible_bbox_mask)[:, :, 4:self.B*5:5].pow(2).sum()
```

**Comment by creotiv** [comment]
You are right. Im at hospital currently, as soon as i will get my laptop i will recheck paper and update the code.
